hw3/scripts/log.txt

```diff
@@ -0,0 +1,58 @@
+## PART 1 ##
+
+#create group mask
+3dmask_tool -input masks/full_mask.sub-*+tlrc.HEAD -frac 1.0 -prefix mask
+
+#compute ttest on word contrast
+3dttest++ -mask mask+tlrc -prefix word -setA 'betas/stats.sub*+tlrc.HEAD[10]'
+
+## PART 2 ##
+
+#extract the relevant row
+for f in blurs/blur_est.sub-*.1D;
+	do
+	1d_tool.py -infile $f -select_rows 5 -write - >> blurs.acf.1D
+done
+
+#avg of the blurs
+1dsum -mean blurs.acf.1D
+
+#use 3dclustsim
+3dClustSim -mask mask+tlrc -acf 0.823891 4.19962 14.369 -both -pthr .05 .01 .001 -athr .1 .05 .025 .01 -iter 2000 -prefix cluster -cmd refit.cmd
```

Comment:
`-iter 2000` is faster and fine for HW, but for real analysis use >5000 (10000 is the default)